compass/landice/tests/humboldt/mesh.py

```diff
@@ -0,0 +1,347 @@
+        check_call(args, logger=logger)
+
+        print('calling interpolate_to_mpasli_grid.py')
+        args = ['interpolate_to_mpasli_grid.py', '-s',
+                'humboldt_1km_2020_04_20.epsg3413.icesheetonly.nc', '-d',
+                'gis_1km_preCull.nc', '-m', 'b', '-t']
+
+        check_call(args, logger=logger)
+
+        print('calling define_cullMask.py')
+        args = ['define_cullMask.py', '-f',
+                'gis_1km_preCull.nc', '-m'
+                'distance', '-d', '50.0']
+
+        check_call(args, logger=logger)
+
+        print('calling set_lat_lon_fields_in_planar_grid.py')
+        args = ['set_lat_lon_fields_in_planar_grid.py', '-f',
+                'gis_1km_preCull.nc', '-p', 'gis-gimp']
+
+        check_call(args, logger=logger)
+
+        print('calling MpasMaskCreator.x')
+        args = ['MpasMaskCreator.x', 'gis_1km_preCull.nc',
+                'humboldt_mask.nc', '-f', 'Humboldt.geojson']
+
+        check_call(args, logger=logger)
+
+        print('culling to geojson file')
+        dsMesh = xarray.open_dataset('gis_1km_preCull.nc')
+        humboldtMask = xarray.open_dataset('humboldt_mask.nc')
+        dsMesh = cull(dsMesh, dsInverse=humboldtMask, logger=logger)
+        write_netcdf(dsMesh, 'humboldt_culled.nc')
+
+        print('Marking horns for culling')
+        args = ['mark_horns_for_culling.py', '-f', 'humboldt_culled.nc']
+        check_call(args, logger=logger)
+
+        print('culling and converting')
+        dsMesh = xarray.open_dataset('humboldt_culled.nc')
+        dsMesh = cull(dsMesh, logger=logger)
+        dsMesh = convert(dsMesh, logger=logger)
+        write_netcdf(dsMesh, 'humboldt_dehorned.nc')
+
+        print('calling create_landice_grid_from_generic_MPAS_grid.py')
+        args = ['create_landice_grid_from_generic_MPAS_grid.py', '-i',
+                'humboldt_dehorned.nc', '-o',
+                'Humboldt_1to10km.nc', '-l', levels, '-v', 'glimmer',
+                '--beta', '--thermal', '--obs', '--diri']
+
+        check_call(args, logger=logger)
+
+        print('calling interpolate_to_mpasli_grid.py')
+        args = ['interpolate_to_mpasli_grid.py', '-s',
+                'humboldt_1km_2020_04_20.epsg3413.icesheetonly.nc',
+                '-d', 'Humboldt_1to10km.nc', '-m', 'b', '-t']
+        check_call(args, logger=logger)
+
+        print('Marking domain boundaries dirichlet')
+        args = ['mark_domain_boundaries_dirichlet.py',
+                '-f', 'Humboldt_1to10km.nc']
+        check_call(args, logger=logger)
+
+        print('calling set_lat_lon_fields_in_planar_grid.py')
+        args = ['set_lat_lon_fields_in_planar_grid.py', '-f',
+                'Humboldt_1to10km.nc', '-p', 'gis-gimp']
+        check_call(args, logger=logger)
+
+        print('creating graph.info')
+        make_graph_file(mesh_filename='Humboldt_1to10km.nc',
+                        graph_filename='graph.info')
+
+    def build_cell_width(self):
```

Comment:
Can you add a docstring for this function?  I think eventually this function could be abstracted as a general mesh gen function.  It's fine to not deal with that in this PR, but make a note of it in the docstring.

Reply:
Added.